core/common/src/main/java/alluxio/underfs/UnderFileSystemFactoryRegistry.java

```diff
@@ -99,10 +105,18 @@ private UnderFileSystemFactoryRegistry() {}
   public static UnderFileSystemFactory find(String path) {
     Preconditions.checkArgument(path != null, "path may not be null");
 
+    final String selectedFactoryFormat =
+        "Selected Under File System Factory implementation {} for path {}";
     for (UnderFileSystemFactory factory : FACTORIES) {
       if (factory.supportsPath(path)) {
-        LOG.debug("Selected Under File System Factory implementation {} for path {}",
-            factory.getClass(), path);
+        LOG.debug(selectedFactoryFormat, factory.getClass(), path);
```

Comment:
I think we should still use two different debug message for line 112 and line 119 to tell the difference: one from service loading and the other is from extension